tests/acceptance/page_objects/base.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+
+
+class BasePage(object):
+    """Base class for PageObjects"""
+    page_name = 'base'
+
+    def __init__(self, browser):
+        self.browser = browser
+        try:
+            self.assert_correct_page()
+        except AssertionError:
+            raise Exception(
+                'This is not the %s page. Current url is %s.' %
+                (self.page_name, self.driver.current_url))
+
+    @property
+    def driver(self):
+        return self.browser.driver
+
+    def assert_correct_page(self):
```

Comment:
Changed this to `pass` rather than throw a `NotImplementedError` to more easily align with Mark's changes to page objects.